packages/api/src/modules/user/controller.ts

```diff
@@ -219,6 +220,7 @@ export class UserController {
     try {
```

Comment:
add an new test to validate this case...
- success upgrade
- error upgrade caused by duplicated name